api/src/main/scala/hmda/api/http/InstitutionsHttpApi.scala

```diff
@@ -40,16 +42,23 @@ trait InstitutionsHttpApi extends InstitutionProtocol with ApiErrorProtocol with
   val institutionsPath =
     path("institutions") {
       val path = "institutions"
-      val institutionsActor = system.actorSelection("/user/institutions")
       timedGet {
         extractRequestContext { ctx =>
-          val ids = institutionIdsFromHeader(ctx)
-          val fInstitutions = (institutionsActor ? GetInstitutionsById(ids)).mapTo[Set[Institution]]
-          onComplete(fInstitutions) {
-            case Success(institutions) =>
-              val wrappedInstitutions = institutions.map(inst => InstitutionWrapper(inst.id.toString, inst.name, inst.status))
-              complete(ToResponseMarshallable(Institutions(wrappedInstitutions)))
-            case Failure(error) => completeWithInternalError(path, error)
+          extractExecutionContext { executor =>
+            implicit val ec: ExecutionContext = executor
+            val ids = institutionIdsFromHeader(ctx)
+            val supervisor = system.actorSelection("/user/supervisor")
```

Comment:
This actor is used in every endpoint... Would it be better to have a single supervisor declaration at the top of this file?